modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/methods_tostr_eq_simple.mustache

```diff
@@ -2,6 +2,10 @@
         """Returns the string representation of the model"""
         return str(self.value)
 
+    def __hash__(self):
+        """Returns the hash value of this object."""
+        return hash(self._data_store['value'])
```

Comment:
Can you change this to `hash(frozenset(self._data_store.items()))` so it is the same as the one in modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/methods_todict_tostr_eq_shared.mustache ? That will reduce the chance of hash collisions.

Reply:
Based on the discussion, I think I can mark this resolved.

Reply:
Yup, no need for this method anymore